### ISSUE TITLE
sbt 2.0.0

### ISSUE THREAD

**Issue by raboof** [raboof/sbt-reproducible-builds#344]
Requires upstream:

* [ ] https://github.com/sbt/sbt-git/pull/256
* [ ] ci-release
* [ ] ... sbt-reproducible-builds :laughing: 

I guess we can live without

**Comment by BillyAutrey** [comment]
I started on `sbt-ci-release`. Because it pulls in plugins via `addSbtPlugin`, as far as I can tell we're forced to require crossbuilds of _those_ plugins as well.

This means we'll need 2.0.0-M2 crossbuilds of:
* `sbt-dynver`
* `sbt-git` (in progress)
* `sbt-pgp` (done with `2.3.0`)
* `sbt-sonatype`.

**Comment by BillyAutrey** [comment]
Side note - this plugin strips timestamps. But `zinc` removes them in 2.0.0, actually to try and create more reproducible builds.

I think some of this plugin's aims are still valuable to have separately (e.g., buildinfo for publication). But I wonder how much this will have to change for 2.0.0?

**Comment by raboof** [comment]
> Side note - this plugin strips timestamps. But `zinc` removes them in 2.0.0, actually to try and create more reproducible builds.
> 
> I think some of this plugin's aims are still valuable to have separately (e.g., buildinfo for publication). But I wonder how much this will have to change for 2.0.0?

I agree! Making the stripping (or perhaps individual strippers?) optional is something I'd like to do in any case (#299), and under sbt 2.x could perhaps even become the default.

**Comment by BillyAutrey** [comment]
> great that you're pushing for this! should we create a tracking issue in sbt-ci-release and link to that here?

Sure, that's a good idea. I can do that.

**Comment by BillyAutrey** [comment]
https://github.com/sbt/sbt-ci-release/issues/314
